packages/astro/src/@types/astro.ts

```diff
@@ -892,6 +892,26 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		excludeMiddleware?: boolean;
+
+		/**
+		 * @docs
+		 * @name build.filterPage
+		 * @type {(pageURL: string) => boolean}
+		 * @default `undefined`
+		 * @version 2.10.0
+		 * @description
+		 * When defined, will be called for every single page about to be generated and gives opportunity to skip the page. When returning `false`, the page matching the finale `pageURL` will be excluded from the static build.
```

Comment:
Some docs wording suggestions, I've tried to make it closer to the format we have in other APIs docs, while adding some extra clarification. 

If I got it right, this should also apply to prerendered pages in SSR, right? That's something I'd like to clarify if true.
```suggestion
* Defines a function that will check which pages should be filtered out from the build while returning `false`.
*
* When defined, its function will be called for every single page about to be generated and gives the opportunity to skip the page. The criteria defined by the user based on `pageURL`, while returning `false`, will exclude any generated pages from the static build (including prerendered pages).
```

Reply:
@Yan-Thomas thank a lot for your suggestions, i applied them right away.